common/formatting/unwrapped_line.h

```diff
@@ -94,9 +94,13 @@ enum class PartitionPolicyEnum {
   // FormatStyle.wrap_spaces when wrapping.
   kAppendFittingSubPartitions,
 
-  // Does optimizations on partition tree and its subpartitions.
-  // Currently it used for reshaping function calls partitions.
-  kOptimalFunctionCallLayout,
+  // Layout Optimizer policies. See its implementation (layout_optimizer.cc)
```

Comment:
`LayoutOptimizer` (no space)

Same in other places that reference the class name.

Reply:
Done